app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -31,13 +31,19 @@
 
   @include unselectable;
 
+  // 100dvh with fallback for old browsers
+  --100dvh: 100vh;
+  @supports (height: 100dvh) {
+    --100dvh: 100dvh;
+  }
+
   .footer-nav-ipad & {
     top: calc(var(--header-offset) + var(--footer-nav-height));
     height: calc(
-      100vh - var(--header-offset, 0px) - var(--footer-nav-height, 0px)
+      var(--100dvh) - var(--header-offset, 0px) - var(--footer-nav-height, 0px)
```

Comment:
Any reason not to simply use `100dvh` for the special iPad case and not  introduce a new custom property? As far as I can tell, this is very much scoped to Safari on iPads.

Reply:
So you're saying we should only use `dvh` for the `.footer-nav-ipad` `height`? And go back to `vh` for the regular `height` entry?

`.footer-nav-ipad` only applies to the PWA view on iPad. The motivation for this PR is problems with regular Safari on iPad. So think we need to use `dvh` in both places.

Reply:
Ah, that makes sense.